packages/crypto/src/MultisigSignature.ts

```diff
@@ -0,0 +1,94 @@
+import Address, { MultisigAddress, utils } from '@helium/address'
+import { verifySignature } from './utils'
+import KeySignature from './KeySignature'
+
+const PUBLIC_KEY_LENGTH = 33
+
+export default class MultisigSignature {
+  public addresses!: Address[]
+
+  public signatures!: KeySignature[]
+
+  constructor(addresses: Address[], signatures: KeySignature[]) {
+    this.addresses = addresses
+    this.signatures = signatures
+  }
+
+  public static create(
```

Comment:
same comment here. If there's some validation for what comprises a "valid" instance of MultisigSignature, it probably belongs in the constructor. and I notice that the other static method in this class is referencing this one, so probably safe to move to constructor

Reply:
Similar to my other response I think this allows different uses. 

The `create` method confirms you have the correct number of addresses and sufficient signatures. If instead you want to just create a `MultisigSignature` with a set list of Addresses and KeySignatures (perhaps because it's partially signed and you're forwarding the payload to the next signer, let's say), then you can use the constructor directly.

Let me know if this still doesn't align with your thinking and happy to discuss further.

Reply:
similar feedback. if the `insufficient signatures` or `wrong number of addresses` validation applies to any instance of a MultisigSignature class, then I'd expect that to happen in the constructor directly. if that assumption is untrue, then maybe something about the method naming should change to not insinuate that `create` is the proper way to instantiate a properly constructed instance

Reply:
Updated per our conversation, please let me know if any further updates needed. 😄